src/server/controllers/user.js

```diff
@@ -8,27 +8,39 @@ const jwtSecret = 'mysecret';
 const register = async (req, res) => {
     const { username, password } = req.body;
 
-    const createdUser = null;
+    const hashedPassword = await bcrypt.hash(password, 10)
+
+    const createdUser = await prisma.user.create({
+        data: {
+            username: username,
+            password: hashedPassword
+        }
+    });
 
     res.json({ data: createdUser });
+    // res.json({ 'User created!': createdUser });
```

Comment:
I liked your comments through out the file  but not sure what this commented code is useful